cluster/gce/upgrade.sh

```diff
@@ -603,7 +617,7 @@ if [[ -z "${STORAGE_MEDIA_TYPE:-}" ]] && [[ "${STORAGE_BACKEND:-}" != "etcd2" ]]
   echo "export STORAGE_MEDIA_TYPE=application/json"
   echo ""
   if [ -t 0 ] && [ -t 1 ]; then
-    read -p "Would you like to continue with the new default, and lose the ability to downgrade to etcd2? [y/N] " confirm
+    read -r -p "Would you like to continue with the new default, and lose the ability to downgrade to etcd2? [y/N] " confirm
```

Comment:
why is the `-r` needed here?

Reply:
refer to https://github.com/koalaman/shellcheck/wiki/SC2162